miniscope_io/stream_daq.py

```diff
@@ -23,12 +24,11 @@
 ok_error = None
 try:
     from miniscope_io.devices.opalkelly import okDev
-
     HAVE_OK = True
 except (ImportError, ModuleNotFoundError) as ok_error:
     module_logger = init_logger('streamDaq')
     module_logger.warning(
-        "Could not import OpalKelly driver, unable to read from FPGA!")
+        "Could not import OpalKelly driver, you can't read from FPGA!\nCheck out Opal Kelly's website for install info\nhttps://docs.opalkelly.com/fpsdk/getting-started/")
```

Comment:
ideally we have vendored all this so people don't need to install anything extra :)

Reply:
I wasn't sure how these worked and misunderstood that adding the specified USB driver made the test go through, but I noticed these are just software modules, and that's not the case.